src/Microsoft.AspNet.Owin/OwinFeatureCollection.cs

```diff
@@ -155,6 +155,16 @@ void IHttpResponseFeature.OnSendingHeaders(Action<object> callback, object state
             register(callback, state);
         }
 
+        void IHttpResponseFeature.OnResponseCompleted(Action<object> callback, object state)
+        {
+            var register = Prop<Action<Action<object>, object>>(OwinConstants.CommonKeys.OnResponseCompleted);
```

Comment:
Yeah, just leave this out of OWIN for now. We're trying to restrict the adapter to community defined keys for now.